src/assets/stylesheets/Modal.scss

```diff
@@ -1,15 +1,18 @@
-@use './rpf_design_system/spacing' as *;
-@use './rpf_design_system/font-size' as *;
-@use './rpf_design_system/font-weight' as *;
-@use './rpf_design_system/line-height' as *;
-@use './rpf_design_system/colours' as *;
-@use './rpf_design_system/mixins' as *;
+@use "./rpf_design_system/spacing" as *;
+@use "./rpf_design_system/font-size" as *;
+@use "./rpf_design_system/font-weight" as *;
+@use "./rpf_design_system/line-height" as *;
+@use "./rpf_design_system/colours" as *;
+@use "./rpf_design_system/mixins" as *;
```

Comment:
I had changed these to single quotes throughout, but if the linter prefers double quotes then feel free to change them back

Reply:
Okay, let's go with what the linter's saying then 👍